modules/deploy/pages/deployment-option/self-hosted/kubernetes/k-production-deployment.adoc

```diff
@@ -98,22 +96,16 @@ spec:
       #licenseSecretRef:
         #name: <secret-name>
         #key: <secret-key>
-    statefulset:
-      extraVolumes: |-
-        - name: redpanda-io-config
-          configMap:
-            name: redpanda-io-config
-      extraVolumeMounts: |-
-        - name: redpanda-io-config
-          mountPath: /etc/redpanda-io-config
-      additionalRedpandaCmdFlags:
-        - "--io-properties-file=/etc/redpanda-io-config/io-config.yaml"
+    #statefulset:
```

Comment:
@chrisseto or @david-yu do you guys have a recommendation of what is the best way to expose these now empty options?

Reply:
Remove the entire statefulset block (Everything after line 99 in this PR).

Reply:
Thank Chris, so that's what I had originally but then noticed that there is the paragraph below in the bullet point list which refers to the `statefulset` so it would be a bit without context otherwise. I'd expect there to be other things one can set here?

Reply:
Oh, I didn't see that. Thank you for pointing it out. I think it'd be best to remove that line, otherwise we could specify `statefulset: {}` here. @JakeSCahill what do you think?